ui/app/serializers/clients/activity.js

```diff
@@ -34,20 +34,22 @@ export default class ActivitySerializer extends ApplicationSerializer {
 
   // for vault usage - vertical bar chart
   flattenByMonths(payload, isNewClients = false) {
+    const sortedPayload = [...payload];
+    sortedPayload.reverse();
```

Comment:
I initially created a method to sort months by date - but this requires iterating through and converting the timestamp to a date object, comparing each and _then_ formatting the timestamp into the `M/yy` format. 

The API always returns a chronological array of months in order from most recent to oldest. So to save computing this in the frontend opted to just reverse this order.